backend/main-service/src/main/java/kkakka/mainservice/coupon/domain/Coupon.java

```diff
@@ -112,4 +112,8 @@ public void deleteCoupon() {
     public boolean isDeleted() {
         return this.isDeleted;
     }
+
+    public boolean isPercentageCoupon() {
+        return this.percentage > 0 || this.percentage != null ? true : false;
```

Comment:
```suggestion
        return this.percentage != null && this.percentage > 0;
```
이렇게 표현할 수 있을 것 같네요ㅎㅎ
`this.percentage > 0` 은 `null` 이 아님이 보장되어야 연산할 수 있는데 지금 순서는 `this.percentage != null` 이 의미가 없겠죠..? 그리고 `null` 이면 앞의 `this.percentage > 0` 을 수행하다가 에러가 발생할거구요..

Reply:
아하 그렇군요 정액할인으로 생성할 시 0으로 입력되는지 null로 넘어가는지 확인이 안되서 넣어놨던건데 순서를 생각하지 못했네요 감사합니다